### PR TITLE
Add translation into Czech (cs-CZ)

### PR DESCRIPTION
**Pull request by mormegil-cz**
I created a first version of Czech translation of the phrases (but I’m not sure about the mechanism in `narrative_builder_factory.cc`).

### CODE REVIEW

**Comment by kevinkreiser** [comment]
this is amazing, thank you so much!

**Comment by kevinkreiser** [comment]
looks like there was a problem installing the locale, i'm sure its something ive messed up with the script that does that so ill take a look and send you the change!

File: locales/cs-CZ.json [hunk 1]
```diff
@@ -0,0 +1,907 @@
+{
+  "posix_locale": "cs-CZ.utf8",
```
Comment (kevinkreiser) [inline locales/cs-CZ.json]: this should be `"cs_CZ.utf8"`. posix locales use an underscore as the separator. if you can make this change, the build will hopefully pass and we can merge this!

**Comment by kevinkreiser** [comment]
looks like there are a few issues with the json as well. ill see if i can make you a patch to apply!

File: locales/cs-CZ.json [hunk 2]
```diff
@@ -0,0 +1,905 @@
...
+      }
+    },
+    "post_transit_connection_destination_verbal": {
+      "phrases": {
+        "0": "Jděte na <CARDINAL_DIRECTION>.",
+        "1": "Jděte na <CARDINAL_DIRECTION> na <STREET_NAMES>.",
+        "2": "Jděte na <CARDINAL_DIRECTION> na <BEGIN_STREET_NAMES>."
+      },
+      "cardinal_directions": ["north", "northeast", "east", "southeast", "south", "southwest", "west", "northwest"],
+      "example_phrases": {
```
Comment (kevinkreiser) [inline locales/cs-CZ.json]: you appear to be missing this: `"empty_street_name_labels": ["chodníku", "cyklostezce", "stezce pro horská kola"],`

**Comment by mormegil-cz** [comment]
I fought a bit with the builder/testing scripts, but I have succeeded at last, it seems.

**Comment by dgearhart** [comment]
Excellent - thank you
I will be adding a new entry for some transit phrases later today - I will add the english phrases as place holders to the cs-CZ.json file.

**Pull request merged**